src/Compilers/CSharp/Portable/Symbols/Source/SourceNamespaceSymbol.AliasesAndUsings.cs

```diff
@@ -23,13 +24,31 @@ public Imports GetImports(CSharpSyntaxNode declarationSyntax, ConsList<TypeSymbo
                 case CompilationUnitSyntax compilationUnit:
                     if (!compilationUnit.Externs.Any() && !compilationUnit.Usings.Any())
                     {
-                        return Imports.Empty;
+                        var result = GetGlobalUsingImports(basesBeingResolved);
```

Comment:
>GetGlobalUsingImports [](start = 37, length = 21)

nit: consider renaming `GetGlobalUsingImports` to GetGlobalImports` or `GetImports` to `GetUsingImports` to maintain naming convention between global and non-global methods (ie. just adding "Global").

Reply:
> consider renaming GetGlobalUsingImports to GetGlobalImportsorGetImportstoGetUsingImports` to maintain naming convention between global and non-global methods (ie. just adding "Global").

There is an important distinction between the two GetGlobalUsingImports builds Imports only from global usings. GetImports takes extern aliases into accout. From my perspective there is no parallel here. The parallel exists only around usings.


---
In reply to: [595733837](https://github.com/dotnet/roslyn/pull/51855#discussion_r595733837) [](ancestors = 595733837)